src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -214,6 +217,22 @@ class WiFiManager
     WiFiNetwork mWantedNetwork{};
     bool mInternalScan{ false };
     uint8_t mRouterSolicitationCounter = 0;
+    bool mSsidFound{ false };
+
+#if CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_ENABLE
+    static constexpr uint32_t kConnectionRecoveryMinIntervalMs = CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_MINIMUM_INTERVAL;
+    static constexpr uint32_t kConnectionRecoveryMaxIntervalMs = CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_MAXIMUM_INTERVAL;
```

Comment:
I think we should add some assert checking if min interval < max interval. Currently nothing prevents user from configuring it like this.

Reply:
static asserts sound great here.